app/views/evidence_hub/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="l-constrained">
   <div class="row">
-    <div class="hero hero--blue">
+    <div class="hero hero--blue styleguide__hero">
```

Comment:
is this a temporary class addition? my assumption is we shouldn't have any styleguide classes on the main layouts

Reply:
I agree with your assumption but we don't have a image at the moment and it makes the evidence hub very ugly. I added on temporary until this project has a image to be displayed. We already request but there is no definition yet.

Reply:
ok, happy for it to be on there for now to make it look a little nicer for demos. Lets try not to forget about it